en/Obsidian Sync/Introduction to Obsidian Sync.md

```diff
@@ -9,7 +9,7 @@ Obsidian Sync is a cloud-based service that lets you store your notes on Obsidia
 
 Learn more about Obsidian Sync:
 
-- If you a new Obsidian Sync user, learn how to [[set up Obsidian Sync]].
+- If you are a new Obsidian Sync user, learn how to [[set up Obsidian Sync]].
```

Comment:
```suggestion
- If you're a new Obsidian Sync user, learn how to [[set up Obsidian Sync]].
```

We're using contractions for the Obsidian docs.

Reply:
@marcusolsson thanks for the feedback regarding the writing style on docs! Applied!